eldev.el

```diff
@@ -3709,7 +3709,16 @@ mode output is restricted to just the version."
   (when parameters
     (signal 'eldev-wrong-command-usage `(t "Unexpected command parameters")))
   (let* ((package     (eldev-package-descriptor))
-         (description (when (fboundp 'package--get-description) (package--get-description package))))
+         (description (when (fboundp 'package--get-description)
+                        (condition-case nil
+                            (package--get-description package)
+                          ;; Description not found in README files and using source directories which triggered error.
+                          ;; Check for commentary in source directory file instead.
+                          ;; Note: using only one (the first) source directory.
+                          (error (let* ((project-dir (or eldev-project-dir (expand-file-name default-directory)))
```

Comment:
Inside command functions `eldev-project-dir` is always set, you don't need `or` here.

Reply:
Also, I wouldn't catch `error`, but only `file-missing`. So that we don't hide unrelated problems if those happen.